src/commands/test/lib/included-files-parser.ts

```diff
@@ -1,8 +1,32 @@
-import { IFileDescriptionJson } from "./test-manifest-reader";
+import { IFileDescriptionJson, ITestCloudManifestJson } from "./test-manifest-reader";
 import * as path from "path";
+import * as pfs from "../../../util/misc/promisfied-fs";
+import { validateXmlFile } from "./xml-util";
+import { out } from "../../../util/interaction";
 
 const invalidCharactersRegexp = /['"!#$%&+^<=>`|]/;
 
+export async function copyIncludedFiles(manifest: ITestCloudManifestJson, include: string[], rootDir: string) {
+  if (!include) {
+    return;
+  }
+
+  const includedFiles = parseIncludedFiles(include, rootDir);
+
+  for (let i = 0; i < includedFiles.length; i++) {
+    const includedFile = includedFiles[i];
+    const copyTarget = path.join(path.dirname(rootDir), includedFile.targetPath);
+
+    if (copyTarget.indexOf(".dll.config") !== -1 && !validateXmlFile(copyTarget)) {
```

Comment:
this test will match "x.dll.config" and ".dll.config.x" and "a.b.dll.config.c.txt". I'd suggest using the lodash function `_.endsWith(".dll.config")` instead - only matches at the end and is more explicit about what you're checking for.

Unless you wanted all the other matches, in which case carry on. 😄